tests/TestCase/AppCakeTestCase.php

```diff
@@ -14,6 +14,7 @@
 use Cake\View\View;
 use Network\View\Helper\NetworkHelper;
 use Cake\TestSuite\TestCase;
+use Cake\TestSuite\IntegrationTestTrait;
```

Comment:
All tests inherit from AppCakeTestCase, please avoid using IntegrationTestTrait in the parent class. Rather move `assertPagesFor404()` into PagesControllerTest.php as it's only used there.

Reply:
assertPagesForErrors() is called within PagesControllerTest and still uses httpClient